package.json

```diff
@@ -13,6 +13,9 @@
     "type": "git",
     "url": "git+https://github.com/stefan-prokop-cz/verify-apple-id-token.git"
   },
+  "files": [
```

Comment:
After you check the styleguide, could you please add some prettify and lint script to the `package.json` to be able to manually prettify all the files or to be able to set husky hooks in the future?

Could you also add the lint script call to the github action build and publish pipeline after the build step?